src/services/k8s/knative.go

```diff
@@ -232,3 +232,36 @@ func Clean(namespace string, config *rest.Config, project *project.Project) erro
 	log.Info("The Knative service was successfully deleted", "host", config.Host, "name", project.Name, "namespace", namespace)
 	return nil
 }
+
+func DomainUpd(kn_domain string, config *rest.Config) error {
+	// Default Knative values
+	configMapName := "config-domain"
+	namespace := "knative-serving"
```

Comment:
We should not be hardcoding this imho, it limits the options in case someone has knative deployed in a different namespace.

Let me see if we can set the right default for the `namespace` flag

Reply:
checking if we can use namespace flag..